packages/docs/blog/2026-02-22-release-26-2-1.md

```
@@ -0,0 +1,22 @@
+authors: MatissJanis
+---
+
+This release contains two critical security fixes for users of SimpleFin, Pluggy or the multi-user feature. If you do not use these features: feel free to skip this version.
+
+<!--truncate-->
+
+**Docker Tag: v26.2.1**
+
+Version: v26.2.1
+
+#### Bugfixes
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)